wheels_0:
  ros__parameters:
    target_speed: 0.35
    rate_seconds: 5

wheels_1:
  ros__parameters:
    target_speed: 0.35
    rate_seconds: 5